codeforboston/vaccinatema
# Enable empty searches.

Confirmed with Harlan on Tuesday; we want to enable empty searches so that users can see all locations (ordered alphabetically instead of by distance).

Currently, the page loads with all locations with availability. In order to see all locations without availability, you have to click the checkbox and enter a ZIP code or address (which is confusing if you don't care about distance from a particular place).

### Test Plan:
* Ensure the page loads with all locations with availability.
* Check the "Show sites that don't have availability" checkbox and search again. See all locations!
* Add a ZIP code. See locations ordered by increasing distance from the ZIP code.
* Clear the search box again. See locations ordered alphabetically again.
